test-browser/tests/solidityUnittests.js

```diff
@@ -33,3 +117,171 @@ function runTests (browser) {
+
+        function beforeEach() public {
+          foo = new SimpleStorage();
+        }
+
+        function initialValueShouldBe100() public returns (bool) {
+          return Assert.equal(foo.get(), 100, "initial value is not correct");
+        }
+
+        function valueIsSet200() public returns (bool) {
+          foo.set(200);
+          return Assert.equal(foo.get(), 200, "value is not 200");
+        }
+
+        function shouldFailForWrongValue200() public returns (bool) {
+          foo.set(300);
+          return Assert.equal(foo.get(), 200, "value is not 200");
+        }
+      }
+        `
+    },
+    'browser/ks2a.sol': {
+      content: `
+      pragma solidity >=0.4.22 <0.6.0;
+      contract Kickstarter {
+          enum State { Started, Completed }
+      
+          struct Project {
+              address owner;
+              string name;
+              uint goal;
+              uint fundsAvailable; // added
+              uint amountContributed; // added
+              State state;
+              mapping(address => uint) funders; // added
+          }
+      
+          Project[] public projects;
+      
+          constructor() public {
+          }
+      
+          function createProject(string memory name, uint goal) public {
+              projects.length++; // new line
+              Project storage project = projects[projects.length - 1];
+              project.name = name;
+              project.goal = goal;
+              project.owner = msg.sender;
+              project.state = State.Started;
+          }
+          
+          function fundProject(uint projectId) payable public {
+          Project storage project = projects[projectId];
+              // require project exists
+              // PLEASE CHECK / or erase
+              // not this: require(projects[projectId].exists, "the project must exist to be funded");
+      
+              // require for... underflow/overflow protection
+              project.funders[msg.sender] += msg.value;
+              project.amountContributed += msg.value;
+              project.fundsAvailable += msg.value;
+      
+              if (project.amountContributed >= project.goal) {
+                  project.state = State.Completed;
+              }
+          }
+          
+          // this function is here because we can't use web3 when using the VM
+          function getContractBalance() public view returns(uint balance) {
+              return address(this).balance;
+          }
+            
+      }
+        `
+    },
+    'browser/ks2b_test.sol': {
+      content: `
+      pragma solidity >=0.4.22 <0.6.0;
+      pragma experimental ABIEncoderV2;
+
+      import "remix_tests.sol"; // this import is automatically injected by Remix.
+      import "remix_accounts.sol";
+      import "./ks2a.sol";
+
+      contract kickstarterTest {
+          enum State { Started, Completed }
+
+          Kickstarter kickstarter;
+          /// #sender: account-0
```

Comment:
custom sender doesn't work with special method like `beforeEach`, `beforeAll` etc. for now.